openfl/component/envoy/envoy.py

```diff
@@ -86,13 +86,15 @@ def _save_data_stream_to_file(data_stream):
     def send_health_check(self):
         """Send health check to the director."""
         logger.info('The health check sender is started.')
+        timeout = DEFAULT_TIMEOUT_IN_SECONDS
         while True:
-            self.director_client.send_health_check(
-                self.name,
-                self.is_experiment_running,
-                DEFAULT_TIMEOUT_IN_SECONDS
+            new_timeout = self.director_client.send_health_check(
+                collaborator_name=self.name,
+                is_experiment_running=self.is_experiment_running,
+                valid_duration=timeout * 2
             )
```

Comment:
Is it really required to change health check timeout?

Reply:
It was refactored.